<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sys_Status_Bar_Wid</class>
 <widget class="QWidget" name="Sys_Status_Bar_Wid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>80</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>85</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget_close" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>6</y>
     <width>791</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_astatxt">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Area state</string>
    </property>
    <property name="text">
     <string>AREA-S</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_status_dbcon">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>no</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>-1</y>
      <width>20</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_mem">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Required memory</string>
    </property>
    <property name="text">
     <string>MEM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mstattxt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Measure state</string>
    </property>
    <property name="text">
     <string>MEA-S</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_errortxt">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Number of occurred errors</string>
    </property>
    <property name="text">
     <string>ERR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_memnumber">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0 b</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Risk state, activate it for performing a risk analysis</string>
    </property>
    <property name="text">
     <string>RISK-S</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_dbcontxt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>State of the database connection</string>
    </property>
    <property name="text">
     <string>DB-CON</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_risk_state">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>30</y>
      <width>16</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>-1</y>
      <width>20</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-1</y>
      <width>20</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>19</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_fpl_calctxt">
       <property name="text">
        <string>FPL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_status_fpl">
       <property name="text">
        <string>no</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_dam_calctxt">
       <property name="text">
        <string>DAM</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_status_dam">
       <property name="text">
        <string>no</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_madm_calctxt">
       <property name="text">
        <string>MADM</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_status_madm">
       <property name="text">
        <string>no</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_hyd_calctxt">
       <property name="text">
        <string>HYD</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_status_hyd">
       <property name="text">
        <string>no</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_risk_calctxt">
       <property name="text">
        <string>RISK</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_status_risk">
       <property name="text">
        <string>no</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_cgpu_calctxt">
       <property name="text">
        <string>C/GPU</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_status_cgpu">
       <property name="text">
        <string>0C 0G</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_warningtxt">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>0</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Number of occured warnings</string>
    </property>
    <property name="text">
     <string>WARN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayout">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>29</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label_status_errornum">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_status_warningnum">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayout_2">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>29</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label_area_state">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_measure_state">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>-10</y>
      <width>20</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_threadtxt">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>0</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Current threads in use</string>
    </property>
    <property name="text">
     <string>THREAD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="my_close">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>0</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
